topostats/validation.py

```diff
@@ -54,7 +54,7 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
             ".topostats",
             error="Invalid value in config for 'file_ext', valid values are '.spm', '.jpk', '.ibw', '.gwy', '.topostats', or '.asd'.",
         ),
-        "loading": {"channel": str},
+        "loading": {"channel": str, "scale": dict},
```

Comment:
I'll have to give this some thought as to how to do it, but it might be worth validating that the values of the dictionary are numbers (typically I think they'll be `float`) so that we capture early any invalid configuration options.